src/js/components/ServiceDetail.js

```diff
@@ -22,8 +31,53 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
     };
 
     this.state = {
-      currentTab: Object.keys(this.tabs_tabs).shift()
+      currentTab: Object.keys(this.tabs_tabs).shift(),
+      progressDetailModalOpen: false,
+      servicePlan: null
     };
+
+    this.store_listeners = [{
+      name: 'servicePlan',
+      events: [
```

Comment:
Disclaimer: I don't see this in the previous PRs, so I'm going to comment here.
#nit Could we have this in one line?